R/pkg/NAMESPACE

```diff
@@ -24,7 +24,8 @@ exportMethods("glm",
               "spark.kmeans",
               "fitted",
               "spark.naiveBayes",
-              "spark.survreg")
+              "spark.survreg",
+              "spark.isotonicRegression")
```

Comment:
Spark MLlib ```IsotonicRegression``` is more similar with R [```pava```](http://www.inside-r.org/packages/cran/Iso/docs/pava) ? Should it be named ```spark.pava``` better?